float64[] error_list
bool mode_of_operation
bool recovery_mode_flag
bool safety_limits_flag
int32[] status
